content/blog/2019/performance-best-practices/index.md

```diff
@@ -0,0 +1,110 @@
+
+Next, when testing Istio's data plane performance at scale, it's important to test not only at increasing requests per second, but also against an increasing number of **concurrent** connections. This is because real-world, high-throughput traffic comes from multiple clients. The [provided scripts](https://github.com/istio/tools/tree/76e3cb2488303316c8511a3ebe9676828c9d4765/perf/benchmark#example-2) let you perform the same load test with any number of concurrent connections, at increasing RPS.
+
+Lastly, our test environment measures requests between two pods, not many. The client pod is [Fortio](http://fortio.org/), which sends traffic to the server pod.
+
+Why test with only two pods? Because scaling up throughput (RPS) and connections (threads) has a greater effect on Envoy's performance than increasing the total size of the service registry — or, the total number of pods and services in the Kubernetes cluster. When the size of the service registry grows, Envoy does have to keep track of more endpoints, and lookup time per request does increase, but by a tiny constant. If you have many services, and this constant becomes a latency concern, Istio provides a [Sidecar resource](/docs/reference/config/networking/v1alpha3/sidecar/), which allows you to limit which services each Envoy knows about.
+
+## 3. Measure with and without proxies
+
+While many Istio features, such as [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication), rely on an Envoy proxy next to an application pod, you can [selectively disable](/docs/setup/kubernetes/additional-setup/sidecar-injection/#disabling-or-updating-the-webhook) sidecar proxy injection for some of your mesh services. As you scale up Istio for production, you may want to incrementally add the sidecar proxy to your workloads.
+
+To that end, the test scripts provide [three different modes](https://github.com/istio/tools/tree/de2ab3e4650a2eab47002928a42fd5616f395dc2/perf/benchmark#run-performance-tests). These modes analyze Istio's performance when a request goes through both the client and server proxies (`both`), just the server proxy (`serveronly`), and neither proxy (`baseline`). We also provide an option to disable [Mixer](/docs/concepts/observability/) (telemetry) during the performance tests, which will provide results in line with the performance we expect when the Mixer V2 work is completed.
+
+## Istio 1.2 Performance
+
+The [README](https://github.com/istio/tools/tree/de2ab3e4650a2eab47002928a42fd5616f395dc2/perf/benchmark#istio-performance-benchmarking) in the [`perf/benchmarking`](https://github.com/istio/tools/tree/0dabae1760a2c1bffb87bc5cb00eaf9285ba720a/perf/benchmark)directory provides instructions for running these performance tests. We also provide instructions to run the [same scripts for the Linkerd data plane](https://github.com/istio/tools/tree/master/perf/benchmark/linkerd); currently, only latency benchmarking is supported for Linkerd.
```

Comment:
You might only want to link to the repo, the README is displayed prominently enough there. The text is a bit confusing because it refers to the scripts for Istio as "performance tests" but to the scripts for Linkerd as "scripts". To address both issues I suggest:
```suggestion
The [performance and benchmarking](https://github.com/istio/tools/tree/de2ab3e4650a2eab47002928a42fd5616f395dc2/perf/benchmark#istio-performance-benchmarking) repository provides the resources and instructions needed to run these performance tests. We also provide instructions to run the [same performance tests for the Linkerd data plane](https://github.com/istio/tools/tree/master/perf/benchmark/linkerd). Currently, only latency benchmarking is supported for Linkerd.
```